docs/_includes/variation-content.html

```diff
@@ -1,34 +1,38 @@
 <section class="content_main">
 
-    <div class="component-header u-mb30">
-        <h1>
-            {{ page.title }}
-        </h1>
-    </div>
+    <h1 class="component-header u-mb30">
+        {{ page.title }}
+    </h1>
 
     {% if page.variation_groups and page.variation_groups.size > 0 %}
     <ul class="m-list m-list__horizontal u-mb30">
         {% for variation_group in page.variation_groups %}
         {% if variation_group.variation_group_name %}
         <li class="m-list_item">
-            <a class="m-list_link" href="#{{ variation_group.variation_group_name | slugify }}">{{ variation_group.variation_group_name }}</a>
+            <a class="m-list_link"
+               href="#{{ variation_group.variation_group_name | slugify }}">
+                {{ variation_group.variation_group_name }}
+            </a>
         </li>
         {% endif %}
         {% endfor %}
     </ul>
     {% endif %}
 
     {% if page.description %}
-        <section id="description" class="u-mb30" class="lead-paragraph">
-            {{ page.description | markdownify }}
-        </section>
+    <section id="description" class="lead-paragraph u-mb30">
+        {{ page.description | markdownify }}
+    </section>
     {% endif %}
 
     {% if page.variation_groups and page.variation_groups.size > 0 %}
     {% for variation_group in page.variation_groups %}
     <section class="o-variation_group" id="{{ variation_group.variation_group_name | slugify }}">
-        {% if variation_group.variation_group_name  and variation_group.variation_group_name != '' %}
-        <h2 class="o-variation_group_title">{{ variation_group.variation_group_name }}</h2>
+
+        {% if variation_group.variation_group_name and variation_group.variation_group_name != '' %}
+        <h2 class="o-variation_group_title">
```

Comment:
Hmmm pages without group titles lack the padding provided by `o-variation_group_title`. This is tricky because both the group title and the group description are optional so I'm not sure what we could anchor the padding to.

<img width="719" alt="Screen Shot 2020-05-18 at 12 44 24 PM" src="https://user-images.githubusercontent.com/1060248/82238574-88f70e00-9905-11ea-9978-68ce5e356a4c.png">

Reply:
Yeah this is why I was asking in chat last week whether we absolutely wanted optional fields, as for each one we have, we exponentially increase the potential layout variations of the page.

Reply:
Pushed up this garbage https://github.com/cfpb/design-system/pull/496/commits/af229b3e032446caaa47693569cee3d5aa86961c